src/ray/raylet/node_manager.cc

```diff
@@ -2524,9 +2524,7 @@ bool NodeManager::FinishAssignedTask(std::shared_ptr<WorkerInterface> worker_ptr
     task = worker.GetAssignedTask();
     // leased_workers_.erase(worker.WorkerId()); // Maybe RAY_CHECK ?
     if (worker.GetAllocatedInstances() != nullptr) {
-        this->cluster_task_manager_->HandleTaskFinished(worker_ptr);
-      // new_resource_scheduler_->FreeLocalTaskResources(worker.GetAllocatedInstances());
-      // worker.ClearAllocatedInstances();
+      this->cluster_task_manager_->HandleTaskFinished(worker_ptr);
```

Comment:
Can you remove the `this->` (just a style thing, we assume `this->` if the name ends with a _)